pkg/controller/controller_utils.go

```diff
@@ -404,14 +409,41 @@ func getPodsPrefix(controllerName string) string {
 }
 
 func (r RealPodControl) CreatePods(namespace string, template *api.PodTemplateSpec, object runtime.Object) error {
-	return r.createPods("", namespace, template, object)
+	return r.createPods("", namespace, template, object, nil)
+}
+
+func (r RealPodControl) CreatePodsWithControllerRef(namespace string, template *api.PodTemplateSpec, controllerObject runtime.Object, controllerGVK unversioned.GroupVersionKind) error {
```

Comment:
maybe a style thing, but I'd probably present this as a function that builds a controllerRef, and a function that accepts one while making pods.